implementations.yaml

```
@@ -74,6 +74,27 @@ kems:
       - name: ntruhrss701
         implementation: clean
         avx2_implementation: avx2
+  ntruprime:
+    version: 0.0.1
```

This can be `0.1.0`, as we intend to keep it somewhat stable.
```suggestion
    version: 0.1.0
```

---

Done